### DISCUSSION TITLE
Prevent people from uploading badges that arent perfect squares

### DISCUSSION THREAD

**Discussion by Blazekickn** [RetroAchievements/RAIntegration#777] [Ideas]
Id argue for not 64x64 but thats just me. Badges that arent uploaded as squares and have to be stretched usually look awful.

**Comment by kdecks** [comment]
In my view this will cause more problems than solutions. 

Players might upload not at 64x64 but square such as 32x32 or 128x128.
They might upload images that are nearly square such as 62x64.
They might upload images that are not square at all, but don't look bad distorted, due to simple subject matter.

**Comment by Blazekickn** [comment]
perfect square doesnt necessarily mean 64x64 just that the height and width match. Its not a huge deal but you have to admit that there is a lot of garbo screencap badges that wouldnt look as bad if it was cropped as a square.

**Comment by kdecks** [comment]
I know it. I wrote this doc: http://docs.retroachievements.org/Badge-and-Icon-Creation/#badge-file-and-size-best-practices
